### PR TITLE
Add image formats specifications & add multistore behaviors for image settings page

### PR DESCRIPTION
**Pull request by LouiseBonnard**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add image file formats specifications & multistore behaviors for image settings page
| Fixed ticket? | Related to https://github.com/PrestaShop/PrestaShop/issues/18518

### CODE REVIEW

**Comment by Progi1984** [comment]
@LouiseBonnard Some questions : 
* Actually, in front office, all uploaded images in backoffice are renamed in .jpeg and converted in a different format (JPEG / PNG) (which is defined in Improve > Design > Images > Images generation options)
image
* **Question : Do i add WEBP in this generation ?**
image

* For logo image, if no logo image is uploaded for Emails & Invoices, the header logo is used. So Webp can't be used in this case. **Do I need a check on format or we exclude web for header logo image ?**

**Comment by LouiseBonnard** [comment]
Hi @Progi1984, thanks for your feedback!

> Do i add WEBP in this generation ?

In my opinion, yes, we should definitely add the possibility to use WEBP as one of the major formats! And since it uses few resources, it should even improve the store's performance, right? As a consequence, a 'WEBP compression' field should be added just below with a default recommendation. Any idea?

> Do I need a check on format or we exclude web for header logo image ?

Yes, this issue has been raised last week about the SVG, cf. https://github.com/PrestaShop/PrestaShop/pull/21627#issuecomment-728207752. What would be the constraints of checking on the image format? It sounds easier to exclude this format for all types of logo while waiting for better support among email clients and files.

**Comment by Progi1984** [comment]
@LouiseBonnard 

1.
Do i add two options like in my screenshot or only one ?
PHP has a quality level to 80 : https://www.php.net/manual/en/function.imagewebp.php

2. 
The support is bad for emails : https://www.caniemail.com/search/?s=webp
TCPDF is not supported WebP images.

So I recommand to exclude WebP for header logo image.

**Comment by LouiseBonnard** [comment]
1. 2 options, thank you, @Progi1984.

2. Unfortunately. We could drop this possibility, right, but we will have to find a solution for the SVG anyway...

**Review by Progi1984 — APPROVED** [review]

**Review by Robin-Fischer-PS — COMMENTED** [review]
@LouiseBonnard A question : What about the upload of image in FO (in Contact us form and in customized product), is this another topic or should it be handled in this spec too ? (in this case, it could be added in the "list of the pages where images can be uploaded")

**Comment by LouiseBonnard** [comment]
Hi @Robin-Fischer-PS, you're right, I could (and will) add the FO behaviors, thanks!

**Comment by LouiseBonnard** [comment]
Hi @Progi1984, I've added the FO behaviors, see https://github.com/PrestaShop/prestashop-specs/pull/179/commits/235f00e2e45af6dc789880173e4f67ff9514834a. Let me know if it sounds alright to you. ;-)

**Comment by marionf** [comment]
@LouiseBonnard I just added the multistore behaviors for the image settings page

**Review by marionf — APPROVED** [review]

**Review by Robin-Fischer-PS — COMMENTED** [review]
OK for me, except for the SVG support (this should be investigated by the dev, for security reasons, if I remember well).

**Comment by marionf** [comment]
@Progi1984 Is this PR ok for you ?

File: broader-topics/image-file-formats.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Favicons are a specific case using precise image formats and it should be addressed later. And the email support for SVG format is too limited for now.
+
+**WEBP.** Decision has been made to first focus on all images upload available in the back office, except for the favicon, the email, and the invoice logos in the Design > Theme & Logo page.
+
+Favicons are a specific case using precise image formats and it should be addressed later. Same for the email and PDF documents that do not enough support WebP images for now.
+
+
+### Front office 
+
+Customers can upload images in two places in the front office: in the contact form and on a customized product page. Images formats allowed are: .png, .jpg, and .gif. Expected behavior should be to **add the .svg format to this list** since one of its great advantages is that it allows resizing with limited loss of quality.
```
Comment (marionf) [inline broader-topics/image-file-formats.md:68]: @PierreRambaud is this pat of the spec is ok for you about security ?

**Comment by Progi1984** [comment]
> @Progi1984 Is this PR ok for you ?

OK for me @marionf

**Review by matthieu-rolland — APPROVED** [review]

**Comment by marionf** [comment]
Thank you @LouiseBonnard :tada:

**Pull request merged**